metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bigcommerce-monit",
+  "version": "1.0.0",
+  "author": "Bigcommerce - Technical Operations <techops@bigcommerce.com>",
+  "license": "Proprietary",
```

Comment:
:confused: Thay module doens't have a `metadata.json` ?

I have no idea what you want me to do here ;)